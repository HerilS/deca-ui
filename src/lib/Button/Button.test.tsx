describe('components/Button', () => {
  it('works', () => {
    const a = 2;
    expect(a).toEqual(2);
  });
});
