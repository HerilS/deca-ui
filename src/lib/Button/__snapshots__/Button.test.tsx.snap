// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Button matches screenshot 1`] = `
<DocumentFragment>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-jsOpSy-color-primary c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-dSKgWl-cv"
    role="button"
  >
    Label
  </button>
</DocumentFragment>
`;

exports[`components/Button renders all colors 1`] = `
<DocumentFragment>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-jsOpSy-color-primary c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-dSKgWl-cv"
    role="button"
  >
    Label
  </button>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-bmmCCE-color-secondary c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-chBTEJ-cv"
    role="button"
  >
    Label
  </button>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-UIZmM-color-success c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-fMjTrh-cv"
    role="button"
  >
    Label
  </button>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-eNqAek-color-warning c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-dKsfPN-cv"
    role="button"
  >
    Label
  </button>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-dodmCG-color-error c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-gURxYt-cv"
    role="button"
  >
    Label
  </button>
</DocumentFragment>
`;

exports[`components/Button renders all sizes 1`] = `
<DocumentFragment>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-jsOpSy-color-primary c-jSUOIM-dPmqWS-size-sm c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-dSKgWl-cv"
    role="button"
  >
    Label
  </button>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-jsOpSy-color-primary c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-dSKgWl-cv"
    role="button"
  >
    Label
  </button>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-jsOpSy-color-primary c-jSUOIM-STxME-size-lg c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-dSKgWl-cv"
    role="button"
  >
    Label
  </button>
</DocumentFragment>
`;

exports[`components/Button renders all variants 1`] = `
<DocumentFragment>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-jsOpSy-color-primary c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-dSKgWl-cv"
    role="button"
  >
    Label
  </button>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-jsOpSy-color-primary c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-bRdhPS-cv"
    role="button"
  >
    Label
  </button>
  <button
    aria-disabled="false"
    class="c-jSUOIM c-jSUOIM-jsOpSy-color-primary c-jSUOIM-gWJSuX-size-md c-jSUOIM-bDGmTT-disabled-false c-jSUOIM-cyBLNz-cv"
    role="button"
  >
    Label
  </button>
</DocumentFragment>
`;
